molecule/default/molecule.yml

```
@@ -1,118 +1,27 @@
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  - name: ubuntu-22-04  # EOL 2027-04-01
-    image: dokken/ubuntu-22.04:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  - name: ubuntu-24-04  # EOL 2029-04-01
-    image: dokken/ubuntu-24.04:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  # rpm
-  - name: almalinux-9  # EOL 2027-05-31
-    image: dokken/almalinux-9:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  - name: rockylinux-9  # 2032-05-31
-    image: dokken/rockylinux-9:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  - name: amazonlinux-2023  # EOL 2029-06-30
-    image: dokken/amazonlinux-2023:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  - name: centos-stream-9  # EOL 2027-05-31
-    image: dokken/centos-stream-9:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  - name: centos-stream-10  # EOL 2030-01-01
-    image: dokken/centos-stream-10:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  - name: fedora-41  # EOL 2025-11-19
+  - name: fedora-41
     image: dokken/fedora-41:latest
     override_command: false
     volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  - name: oraclelinux-9  # EOL 2032-06-30
-    image: dokken/oraclelinux-9:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-    cgroupns_mode: host
-    privileged: true
-  # suse
-  - name: opensuse-leap-15.6  # EOL 2025-12-01
-    image: dokken/opensuse-leap-15.6:latest
-    override_command: false
-    volumes:
-      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
+      - /home/saksham/packaging-java/target:/var/tmp/target
```

```suggestion
      - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
```
Cf https://ci.jenkins.io/job/Packaging/job/packaging/job/PR-536/2/pipeline-overview/?start-byte=0&selected-node=50#log-50-451 and the error 4 lines later:
> '/var/tmp/target/rpm' is not a directory